driver/normalizer/normalizer.go

```diff
@@ -87,20 +87,32 @@ var Normalizers = []Mapping{
 		CasesObj("case",
 			// common
 			Obj{
-				"importKind": String("value"),
 				"source":     Var("path"),
 			},
 			Objs{
 				// namespace
 				{
+					"importKind": String("value"),
 					"specifiers": ArrWith(Var("names"), Obj{
+						//uast.KeyType: Var("spec_type"),
 						uast.KeyType: String("ImportNamespaceSpecifier"),
 						uast.KeyPos:  Var("local_pos"),
 						"local":      Var("local"),
 					}),
 				},
+				// specific type
+				{
+					"importKind": String("type"),
+					"specifiers": ArrWith(Var("names"), Obj{
+						uast.KeyType: String("uast:Alias"),
+						uast.KeyPos: Var("local_pos"),
+						"Name": Var("local"),
+						"Node": AnyNode(nil), // Hardcoded node "." on the elevated uast:Alias
```

Comment:
So it shouldn't be `Any` in this case. You should specify what kind of node is here (for the case that you found similar to "." in Alias).

Reply:
Changed.